narwhal/worker/src/handlers.rs

```diff
@@ -77,6 +78,43 @@ impl<V: TransactionValidator> WorkerToWorker for WorkerReceiverHandler<V> {
 
         Ok(anemo::Response::new(RequestBatchResponse { batch }))
     }
+
+    async fn request_batches(
+        &self,
+        request: anemo::Request<RequestBatchesRequest>,
+    ) -> Result<anemo::Response<RequestBatchesResponse>, anemo::rpc::Status> {
+        const MAX_REQUEST_BATCHES_RESPONSE_SIZE: usize = 6_000_000;
+        const MAX_BATCH_DIGESTS: usize = 100;
+        let mut remaining_batch_digests = Vec::new();
```

Comment:
should `remaining_batch_digests` also include the digests that were truncated because they exceed `MAX_BATCH_DIGESTS`?

Reply:
Yes! Included it as part of the remaining digests now